data/pokemon/base_stats/abra.asm

```diff
@@ -12,7 +12,7 @@
 	db 20 ; step cycles to hatch
 	db 5 ; unknown 2
 	INCBIN "gfx/pokemon/abra/front.dimensions"
-	db 0, 0, 0, 0 ; padding
+	dw 0, 0 ; former front/back pics
```

Comment:
This reads weird to me. Maybe say `; unused (front/backpics in generation 1)` or some form that mentions _when_ they were used.

Reply:
That seems to make far more sense description-wise.

Reply:
Sure. I'm using "beta" rather than "Gen 1" though, IMO that's more applicable to pokecrystal since spaceworld still used those pointers.